api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -601,6 +601,11 @@ message DeleteAllApplicationServersRequest {
   string Namespace = 1 [(gogoproto.jsontag) = "namespace"];
 }
 
+// ListOfStrings is a generic list of strings.
+message ListOfStrings {
```

Comment:
Use wrappers.StringValues instead of duplicating it?

Reply:
Ahhh, thank you. I was looking for something like that!